CHANGELOG-5.x.md

```diff
@@ -2,17 +2,22 @@
 
 * Decoupled test execution and reporting from PHPUnit
 * Custom reporters implementing TestListener are no longer supported and must be converted to Extensions
-* Tests of all formats are reported as useless if they perform no assertions and reports_useless_tests setting is enabled
+* Tests of all formats are reported as useless if they perform no assertions and `reports_useless_tests` setting is enabled
 * Added path_coverage setting to enable path and branch coverage #6158 by @s0lus
-* Added optional value to fail-fast option (#6275) by #Verest
-* Removed JSON and TAP loggers
+* Added optional value to `fail-fast` option (#6275) by #Verest
+* Removed `JSON` and `TAP` loggers
 * Removed code coverage blacklist functionality
 * Removed deprecated class aliases
   - Codeception\TestCase\Test
   - Codeception\Platform\Group
   - Codeception\Platform\Group
   - Codeception\TestCase
-* Removed settings disallow_test_output, log_incomplete_skipped and report_useless_tests.
+* Removed `generate:cept` command
+* Removed settings `disallow_test_output`, `log_incomplete_skipped` and `report_useless_tests`.
```

Comment:
will `report_useless_tests` be removed after all?

Reply:
No